akka-samples/akka-sample-cluster-java/src/main/resources/application.conf

```diff
@@ -15,7 +16,14 @@ akka {
       "akka.tcp://ClusterSystem@127.0.0.1:2551",
       "akka.tcp://ClusterSystem@127.0.0.1:2552"]
 
+    //#snippet
+    // excluded from snippet
```

Comment:
It is actually, but I'll better stick to the `#` syntax here indeed :) 
See: https://github.com/typesafehub/config#features-of-hocon